src/OrchardCore.Modules/OrchardCore.Localization/Views/LocalizationSettings.Edit.cshtml

```diff
@@ -0,0 +1,77 @@
+@model LocalizationSettingsViewModel
+
+<p class="alert alert-warning">@T["The current tenant will be reloaded when the settings are saved."]</p>
+
+@{ 
+    // ['fr-FR', 'en-US']
+    var options = "['" + String.Join("','", Model.Cultures.Where(x => x.Supported).Select(x => x.CultureInfo.Name)) + "']";
+    var defaultCulture = Model.Cultures.Where(x => x.IsDefault).FirstOrDefault().CultureInfo.Name;
+    var selectedCulture = Model.Cultures.First().CultureInfo.Name;
+}
+
+<script asp-src="~/OrchardCore.Localization/Scripts/optionsEditor.min.js" debug-src="~/OrchardCore.Localization/Scripts/optionsEditor.js" asp-name="optionsEditor" at="Foot" depends-on="vuejs, vuemultiselect"></script>
+<script asp-src="https://vuejs.org/js/vue.min.js" debug-src="https://vuejs.org/js/vue.js" asp-name="vuejs" at="Foot"></script>
+
+<script at="Foot">
+    initializeOptionsEditor(document.getElementById('@Html.IdFor(m => m)'), @Html.Raw(options), '@defaultCulture', '@selectedCulture');
+</script>
+
+<script type="text/x-template" id="options-table">
+    <fieldset class="form-group">
+        <table class="table">
+            <thead class="">
+                <tr>
+                    <th scope="col">@T["Culture"]</th>
+                    <th scope="col" colspan="3">@T["Default Culture"]</th>
```

Comment:
Why col-span 3?!! The tables contains 3 including culture name. I'm sure it's radio button style 😊